capstone/src/main/webapp/script.js

```diff
@@ -317,33 +319,70 @@ async function averagePrecinctSentiment(policePrecinct) {
     }
   }
   const sentimentAverage = sentimentCount/list.length;
-  return getSentimentColor(sentimentAverage);
+  return getSentimentColor(sentimentAverage, policePrecinct);
 }
-
-function getSentimentColor(averageFeelings) {
+//* *colors precinct on map based on strong dislike to strong like */
+function getSentimentColor(averageFeelings, policePrecinct) {
+  let precinctColor = '#228B22';
   if (Math.round(averageFeelings) == 5) {
-    return '#165B33';
+    precinctColor = '#165B33';
   } else if (Math.round(averageFeelings) == 4) {
-    return '#146B3A';
+    precinctColor = '#146B3A';
   } else if (Math.round(averageFeelings) == 3) {
-    return '#F8B229';
+    precinctColor = '#F8B229';
   } else if (Math.round(averageFeelings) == 2) {
-    return '#EA4630';
+    precinctColor = '#EA4630';
   } else {
-    return '#BB2528';
+    precinctColor = '#BB2528';
   }
-}
+  const precinctLayer = mapAndSelection.map;
 
+    precinctLayer.overrideStyle(precinctLayer.getFeatureById(
+        searchPrecinctsByStation(policePrecinct)),
+    {fillColor: precinctColor, fillOpacity: 0.9});
+}
+//* *global variable for precinctLayer data layer and checkbox selection */
 const mapAndSelection = {};
+//**following two functions are only used in index.html */
 /* eslint-disable no-unused-vars */
+//**unchecks boxes when new checkbox is checked */
 function onlyOne(checkbox) {
   const checkboxes = document.getElementsByName('check');
   checkboxes.forEach((item) => {
-    if (item !== checkbox) item.checked = false;
+    if (item !== checkbox) {
+      item.checked = false;
+    }
   });
-  mapAndSelection.selection = checkbox.id;
-  drawCheckboxLayer();
+
+  if (mapAndSelection.selection == checkbox.id) {
+    mapAndSelection.selection = 'noneSelected';
+    fixMap();
+  } else {
+    mapAndSelection.selection = checkbox.id;
+    drawCheckboxLayer();
+  }
 }
+//* *goes to log in page if user is not logged in */
+async function showStats() {
+  const logStatus = await fetch('/status');
+  const status = await logStatus.json();
+  console.log(status);
+  if (status == true) {
+    window.location.replace('statistics.html');
+  } else {
+    window.location.replace('/login');
+  }
+}
+
+/* eslint-enable no-unused-vars */
+//* *erases precinct coloring after checkbox is unselected */
```

Comment:
Same comment generally for all your comments. :) I'll avoid tagging each one. One line comments generally just use `//`

For multi-line comments, here's how they should look:
https://google.github.io/styleguide/jsguide.html#formatting-block-comment-style

Reply:
I changed the format for my comments for everything but this because it needs to be in that format to make es-lint ignore that rule